### PR TITLE
variable height for modal

### PR DESCRIPTION
**Pull request by thewillhuang**

### CODE REVIEW

File: modal/sandbox/sandbox.jsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Component extends React.Component {
           closeOnOverlayClicked
           show={this.state.show}
           closeModal={() => this.setState({ show: false })}
-          size='large'
+          size='medium'
```
Comment (ben-burlingham) [inline modal/sandbox/sandbox.jsx]: This should be enumerated somehow.

Also, the 2.0 version of this modal should remove the need for sizing...other Particles would achieve this by overriding styles and inserting content in slots.

@Marswang92 what's the status on the Modal fixes?

File: modal/src/Modal.jsx [hunk 2]
```diff
@@ -75,7 +76,7 @@ class Modal extends Component {
     );
 
     return show ? (
-      <div className={styles.container}>
+      <div className={styles.container} style={{ ...{ height: this.props.height } }}>
```
Comment (ben-burlingham) [inline modal/src/Modal.jsx]: This should be achieved by overriding styles.

Comment (thewillhuang) [reply modal/src/Modal.jsx]: how does overriding sytles work?

Comment (thewillhuang) [reply modal/src/Modal.jsx]: actually, I think using the style composer isn't quite as good as using a prop that would just override. it's obvious what height would do. but not obvious what stylesheet would do.

Comment (ben-burlingham) [reply modal/src/Modal.jsx]: Adding props to override individual styles doesn't scale that well.

The stylesheet composer allows you to override any part of the stylesheet, including height of any container.  

The upcoming docs will expose the base stylesheets for each particle, so the dev (or UX designer) can see what to override.  This allows us to work around `application.css` with its `!importants` and also to theme any component to fit any product :)

Comment (thewillhuang) [reply modal/src/Modal.jsx]: okay, so perhaps this pr isn't needed

Comment (thewillhuang) [reply modal/src/Modal.jsx]: great point

**Comment by ben-burlingham** [comment]
@thewillhuang Mars' new modal https://github.com/procore/particles/pull/365

**Pull request closed without merging**